tests/test-integration/pom.xml

```diff
@@ -176,5 +177,10 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <version>1.2.3</version>
```

Comment:
Wouldn't this conflict with jetty-slf4j-impl ?

Reply:
Oh yeah I agree but I removed my tests from this branch